osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -148,11 +153,28 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
         {
             base.LogoArriving(logo, resuming);
 
-            logo.ScaleTo(new Vector2(0.15f), 300, Easing.In);
-            logo.MoveTo(new Vector2(0.5f), 300, Easing.In);
+            const double duration = 300;
+
+            if (!resuming)
+            {
+                logo.MoveTo(new Vector2(0.5f), duration, Easing.In);
+            }
+
+            logo.ScaleTo(new Vector2(0.15f), duration, Easing.In);
             logo.FadeIn(350);
 
-            logo.Delay(resuming ? 0 : 500).MoveToOffset(new Vector2(0, -0.24f), 500, Easing.InOutExpo);
+            content.SetLogo(logo, 1.0f, 500, Easing.InOutExpo);
```

Comment:
this duration should probably match the `resuming ?` one below. currently resuming looks wrong.